src/components/LanguageEditor/LanguageEditor.html

```diff
@@ -1,4 +1,29 @@
 <form class=language-editor>
+  <label>
+    <span>Abbreviation</span>
+    <div>
+      <input
+        inputmode=text
+        name=abbreviation
+        placeholder='e.g. chiti'
```

Comment:
Add a `pattern` attribute to ensure that the data entered in the field is formatted correctly.

```suggestion
        pattern=[(a-z)|(A-Z)|(0-9)]+
        placeholder='e.g. chiti'
```